app/src/main/res/layout/activity_item_list.xml

```diff
@@ -126,6 +126,17 @@
                         android:layout_height="wrap_content"
                         android:text="FILTER"></Button>
                 </LinearLayout>
+
+                <Button
+                    android:id="@+id/resetButton"
+                    android:layout_width="61dp"
+                    android:layout_height="36dp"
+                    android:text="↻"
```

Comment:
nitpick but could the reset thing j be part of the filter fragment/dialog?

Reply:
I will make it part of the filter fragment after the half way